src/robusta/core/playbooks/internal/ai_integration.py

```
@@ -385,6 +386,45 @@ def holmes_chat(event: ExecutionBaseEvent, params: HolmesChatParams):
+    (e.g. add new fields) without requiring runner changes.
+    """
+    holmes_url = HolmesDiscovery.find_holmes_url(params.holmes_url)
+    if not holmes_url:
+        raise ActionException(
+            ErrorCodes.HOLMES_DISCOVERY_FAILED,
+            "Robusta couldn't connect to the Holmes client.",
+        )
+
+    try:
+        params_dict = params.dict(exclude={"holmes_url", "model"})
+        result = requests.post(
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)